script.js

```diff
@@ -1,12 +1,30 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+let colorlist = ['green', 'pink', 'turquoise', 'red']
 
 function setup() {
   createCanvas(windowWidth, windowHeight);
-    background(255);
+    background(0);
 }
 
 function draw() {
-  noStroke();
-  fill(random(colorlist));
-  ellipse(mouseX, mouseY, 25, 25);
+  // line(20, 10, mouseX, mouseY);
+  // stroke(random(colorlist));
+  // strokeWieght(1);
+  strokeWeight(1);
+  stroke(random(255), random(200), random(200), 120);
+  line(mouseX, mouseY, random(width), random(height));
+  line(mouseX, mouseY, random(width), random(height));
+  line(mouseX, mouseY, random(width), random(height));
+
+  // gill lines stationary
```

Comment:
This is a small lab repo so it's not a big deal, but in general you can just remove commented code! 

Remember that if you commit early and often, you will have old versions of your code in the github history, which means you can always go back to see the old code (no need to comment and save).